drake/systems/onlinePlanning/Util/simulateEuler.m

```diff
@@ -0,0 +1,31 @@
+function xtrajSim = simulateEuler(sysCl,tend,x0,dt,sys_type,xp,up)
```

Comment:
what?  no way.  you can call our existing simulate and set the solver to ode1.  don't duplicate code here...